articles/Powershell_guide.md

```diff
@@ -0,0 +1,162 @@
+# Learning  Powershell
+
+## PowerShell is a command-line shell and scripting language that aims to help administrators and power-users rapidly automate tasks.
+With the help of these three cmdlets, you can find, explore, and leverage the others to perform a wide variety of tasks.
+1. **Get-Command** – this cmdlet can be used to display available commands.
+``` powershell
+S /root> Get-Command *process*
+
+CommandType     Name                                               Version    Source
+-----------     ----                                               -------    ------
+Cmdlet          Debug-Process                                      7.0.0.0    Microsoft.PowerShell.Management
+Cmdlet          Enter-PSHostProcess                                7.0.0.0    Microsoft.PowerShell.Core
+Cmdlet          Exit-PSHostProcess                                 7.0.0.0    Microsoft.PowerShell.Core
+Cmdlet          Get-Process                                        7.0.0.0    Microsoft.PowerShell.Management
+Cmdlet          Get-PSHostProcessInfo                              7.0.0.0    Microsoft.PowerShell.Core
+Cmdlet          Start-Process                                      7.0.0.0    Microsoft.PowerShell.Management
+Cmdlet          Stop-Process                                       7.0.0.0    Microsoft.PowerShell.Management
+Cmdlet          Wait-Process                                       7.0.0.0    Microsoft.PowerShell.Management
+
+PS /root>
+```
+2. **Get-Help** - It provides information about what a cmdlet does, and how it can be used.
+```powershell
+PS /root> Get-Help Start-Process
+
+NAME
+    Start-Process
+    
+SYNTAX
+    Start-Process [-FilePath] <string> [[-ArgumentList] <string[]>] [-Credential <pscredential>] [-WorkingDirectory <string>] [-LoadUserProfile] [-NoNewWindow] [-P
+    assThru] [-RedirectStandardError <string>] [-RedirectStandardInput <string>] [-RedirectStandardOutput <string>] [-WindowStyle {Normal | Hidden | Minimized | Ma
+    ximized}] [-Wait] [-UseNewEnvironment] [-WhatIf] [-Confirm] [<CommonParameters>]
+    
+    Start-Process [-FilePath] <string> [[-ArgumentList] <string[]>] [-WorkingDirectory <string>] [-PassThru] [-Verb <string>] [-WindowStyle {Normal | Hidden | Mini
+    mized | Maximized}] [-Wait] [-WhatIf] [-Confirm] [<CommonParameters>]
+    
+
+ALIASES
+    saps
+    
+
+REMARKS
+    Get-Help cannot find the Help files for this cmdlet on this computer. It is displaying only partial help.
+        -- To download and install Help files for the module that includes this cmdlet, use Update-Help.
+        -- To view the Help topic for this cmdlet online, type: "Get-Help Start-Process -Online" or
+           go to https://go.microsoft.com/fwlink/?LinkID=2097141.
+
+
+PS /root>
+```
+3. **Get-Member** - PowerShell deals with objects. Get-Member serves to help you identify what type of objects you are dealing with when using PowerShell and also what properties and methods are available to that object.
```

Comment:
You could also add content about the difference between a `bash` and `powershell` as in the former is based on text and the latter on objects; perhaps explain this with examples